config/webpack.config.dev.js

```diff
@@ -74,6 +74,12 @@ module.exports = {
       // https://www.smashingmagazine.com/2016/08/a-glimpse-into-the-future-with-react-native-for-web/
       'react-native': 'react-native-web',
       'ovirtapi': `${paths.appSrc}/ovirtapi/index.js`,
+      'utils': `${paths.appSrc}/utils/index.js`,
```

Comment:
I suggest moving our aliases to `webpack.config.commons.js` as a function and spreading it here.  That way we can have the alias defs in 1 files instead of 2.  And we can drop the `react-native` alias, I don't think we'll be using that anytime soon.

````js
  alias: {
    ...commonAliases()
  }
````

and in `webpack.config.commons.js`:
````js
exports.commonAliases = function () {
  return [
    'app-intl': `${paths.appSrc}/intl/index.js`,
    'reducers': `${paths.appSrc}/reducers`,
    'actions': `${paths.appSrc}/actions/index.js`,
    'helpers': `${paths.appSrc}/helpers.js`,
    'app-constants': `${paths.appSrc}/constants/index.js`
  ]
}
````